troposphere/static/js/components/images/detail/header/HeaderView.react.js

```diff
@@ -22,6 +23,14 @@ export default React.createClass({
         });
     },
 
+    showLaunchModal: function(e) {
+        modals.InstanceModals.launch({
+            image: this.props.image,
+            initialView: "BASIC_VIEW"
+        });
+        trackAction("launched-from-image-detail", {});
```

Comment:
The intent of `trackAction("launched-from-image-detail")` was to show the usage of the Image Detail page within the Image Catalog. So, if this was pulled into a context where there is a launch and it's not from the Image Detail, we should remove this. 

`trackAction` is meant as a way to leverage the Intercom API and reflect outward the usage of different elements within the application. Another way to put it, these are meant to answer questions like "does anyone use _<Feature X>_?"